src/index.js

```diff
@@ -1,37 +1,29 @@
-import "core-js/stable";
-import swaggerDocs from './docs';
-import swaggerUI from 'swagger-ui-express';
-import "regenerator-runtime/runtime";
-import DB from "./database";
 import express from "express";
-import routes from "./routes/index";
-
-import "dotenv/config"; // Now, the "process.env" object's properties will include those from the .env file
-
-DB.authenticate()
-  .then(() => {
-    console.log("Database Connected");
-  })
-  .catch((err) => {
-    console.log("Database unable to connect");
-    console.error(err);
-  });
-
-const port = process.env.PORT || 5000;
-
+import "dotenv/config";
+import cors from "cors";
+import http from "http";
+import swaggerUi from "swagger-ui-express";
+// @ts-nocheck
+import sequelize from "./database";
+import userRouter from "../src/routes/user.routes";
+import swaggerDocument from "./docs";
 
 const app = express();
-
+app.use(cors());
 app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use("/api/v1/user/", userRouter);
+app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(swaggerDocument));
 
-app.use("/api", routes);
-routes.use('/api-docs', swaggerUI.serve, swaggerUI.setup(swaggerDocs));
-app.get("/", async (req, res) => {
-  res.send({
-    message: "Hello World!",
-  });
+app.get("/", (req, res) => {
+  res.send("Welcome to Winner's APIs");
 });
 
-app.listen(port, () => {
-  console.log("Server has started!");
-});
+const port = process.env.PORT || 8001,
+  //creating server
+  server = http.createServer(app).listen(port, async () => {
```

Comment:
@Tetoalia I can see you tried to change the index file have you agreed with the others?
we are using express not http